integrations/elasticsearch/ops_agent_metadata.yaml

```diff
@@ -298,3 +298,24 @@ platforms:
     kind: GAUGE
     labels:
     - state
+  - name: workload.googleapis.com/elasticsearch.cluster.in_flight_fetch
```

Comment:
This should also result in an update to the metrics_minimum_supported version as these default metrics will not be collected as part of the current version

Reply:
https://github.com/GoogleCloudPlatform/ops-agent/pull/1279 shall update the version, after the PR merged in, we will rerun the script to re-generate this content PR

Reply:
These metrics were apparently already collected since 2.21.0, but undocumented.